### DISCUSSION TITLE
Saves that don't work for some reason

### DISCUSSION THREAD

**Discussion by Nerago-Fikuxu** [bucanero/apollo-ps4#123] [General]
Hey there, I tried using some saves of Mortal Kombat X and Mortal Kombat 11 I Googled for(forums and such, I don't know if it is safe to put links to it here), but I could not get them to work, I copied them to the place they should be on the internal HDD and resigned them with Apollo but nothing happened/they were not recognized/they gave errors(tried this long ago, so I don't remember exactly), some of the sites seem to have links to paid save wizards, so I wonder: 
PS4 that are no jailbroken can export saves with those wizards and resign on other accounts ?
Is there anything else to do with those saves for them to work(like using save wizard codes on apollo somehow ?)
I got one save to work once, which was the Crash Team Racing Nitro-Fueled which I got from a tutorial that taught how to use apollo to resign saves and such, which is why I found it weird that the other games asves didn't work.

**Comment by bucanero** [comment]
Saves need to match the correct keystone or they won't work. Also saves must match the correct Title-ID and user account. 
Many fake pkgs have wrong keystones and saves won't work with original games. Keystones can be replaced and swapped but you need to have the keystone files.
